presto-docs/src/main/sphinx/admin/web-interface.rst

```diff
@@ -16,7 +28,30 @@ running queries.
 When the server is configured to use HTTPS, a :doc:`password authenticator
 </develop/password-authenticator>` such as :doc:`LDAP </security/ldap>` or
 :doc:`password file </security/password-file>` must be configured in order to
-use the Web UI.
+use the Web UI. In these cases, the authentication type is automatically
+defaulted to the required ``form`` value. It triggers the Web UI to display a
+login form that accepts the relevant username and password and verifies against
+the correct authentication backend.
+
+If you require Web UI to be accessible to anyone, you can set the authentication
+type to ``fixed`` and configure the ``web-ui.user``. As a result any interaction
+in the Web UI, such as killing a running query, is logged with the configured
+user.
+
+The authentication type also supports usage of certificate authentication
+(``certificate``), :doc:`Kerberos authentication </security/server>`
+(``kerberos``), and JWT authentication (``jwt``).
+
+After successful login a user has access to all information of queries submitted
+to Presto under the same username.
```

Comment:
I would state this as something like ".. user can view and kill their own queries. Query of other users may be visible if the user has permissions to view other users queries."

Reply:
Great... I wasn't sure how it works..